1.0-rc.1.md

```diff
@@ -0,0 +1,101 @@
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-1.0-rc.1.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-1.0-rc.1.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-1.0-rc.1.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- It is not possible to upgrade directly to this version of CockroachDB from a version older than `beta-20170413`. Upgrade any older clusters to that version first. [#15228](https://github.com/cockroachdb/cockroach/pull/15228)
+- The `cockroach start` command no longer recognizes the `COCKROACH_INSECURE` and `COCKROACH_CERTS_DIR` environment variables which are intended to be client-only. Use the `--insecure` and `--certs-dir` command-line flags instead. [#15196](https://github.com/cockroachdb/cockroach/pull/15196) [#15288](https://github.com/cockroachdb/cockroach/pull/15288)
+- Older certificate-related command-line flags have been removed; use `--certs-dir` instead. [#15241](https://github.com/cockroachdb/cockroach/pull/15241)
+- Various environment variables have been removed in favor of the new `CLUSTER SETTING` commands described below.
+- The undocumented `SET SYNTAX` statement and the "modern" syntax variant have been removed. [#15064](https://github.com/cockroachdb/cockroach/pull/15064)
+
+### General Changes
+
+- Anonymous usage statistics are now reported by default. To opt out, set `COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING=true` in the environment when starting the server or use `SET CLUSTER SETTING diagnostics.reporting.enabled = false` in the SQL shell for a running cluster. [#14901](https://github.com/cockroachdb/cockroach/pull/14901)
+
+### Build Changes
+
+- CockroachDB now builds on Windows. Windows support is currently experimental and is recommended only for development, not production.
+- Building from source on macOS 10.11 works again. [#15168](https://github.com/cockroachdb/cockroach/pull/15168)
+- Building from source on older CPUs (without SSE4.2) works again. [#15247](https://github.com/cockroachdb/cockroach/pull/15247)
+
+### SQL Language Changes
+
+- New commands `SET CLUSTER SETTING` and `SHOW ALL CLUSTER SETTINGS` can now be used to change certain cluster-wide settings. The following cluster settings are available in this release:
+  - `diagnostics.reporting.enabled`
+  - `enterprise.enabled`
```

Comment:
Not sure if his change got in before the release candidate was picked, but @danhhz tells me that this setting is being removed. I'll remove it from the release notes, in any case.